.gitignore

```diff
@@ -16,3 +16,6 @@ node/package-lock.json
 
 # macOS
 .DS_Store
+
+# vsCode
+.vscode
```

Comment:
do you mind if we include this in the .gitignore so I can mess with my settings.json?